.editorconfig

```diff
@@ -0,0 +1,12 @@
+# https://editorconfig.org/
+root = true
+# Conservatively avoid changing defaults for other file types, e.g. raw json files for test cases,
+# Makefiles, etc.
+[*.{cpp,h,md}]
+charset                  = utf-8
```

Comment:
For code, we enforce ASCII as the character set. You *may* change the encoding if you want.

Reply:
That doesn't seem to be an option. utf-8 is the closest to ascii, I think? From https://editorconfig.org/

> charset: set to latin1, utf-8, utf-8-bom, utf-16be or utf-16le to control the character set.